scripts/ci-e2e.sh

```diff
@@ -83,7 +86,7 @@ make test-e2e ARTIFACTS=$ARTIFACTS
 test_status="${?}"
 
 # If Boskos is being used then release the AWS account back to Boskos.
-[ -z "${BOSKOS_HOST:-}" ] || hack/checkin_account.py
+[ -z "${BOSKOS_HOST:-}" ] || hack/boskos.py --release
```

Comment:
This needs to be something like 

`[ -z "${BOSKOS_HOST:-}" ] || python3 -u hack/boskos.py --release`